docs/general/ad-filtering/filter-policy.md

```diff
@@ -3,123 +3,289 @@ title: AdGuard filter policy
+
+- Requests for permission to receive push notifications
+- Popups and forms for subscribing to news, promotions, and various events, including third-party channels for receiving them (such as Google News, Telegram)
+- Popups that encourage users to disable ad blocker and violate user’s privacy (at the discretion of the filter developers)
+- Other types of popups that may annoy users (at the discretion of filter developers)
 
 **Limitations and Exceptions**
 
-If a rule is subject to the list of limitations described below, then it won’t be added to this filter.
+- Push notifications are only blocked on sites where they are not used for practical purposes. For example, in email web clients or tools used for work purposes, such notifications will not be blocked
+- Some popups that do not fall into the categories described above but still interfere with the user’s experience may be also blocked. For example, registration prompts on a site or popups that introduce the site’s features. The decision is made by filter developers
+- Content access measures that ask the user to pay to access the content must not be circumvented
+
+#### AdGuard Mobile App Banners filter
 
-- Rules that would reduce user security levels will be removed.
-- Rules that cause problems with website functionality will be removed.
-- Website-specific rules will be added only if a website has sufficient traffic. Traffic is determined by open statistics (if available) or by other means, such as followers on social media. A website's traffic is considered sufficient when it has 100 thousand visits per month. We will consider adding a rule for a website that's not that popular, but the final decision is up to the filter list maintainer.
+This is a filter that blocks banners and popups that encourage visitors to install mobile applications.
 
-## Social Media filter
+**Limitations and Exceptions**
+
+Banners located in the headers or in the menus of websites are not blocked if they are not animated and do not occupy a significant portion of usable space. If a banner is located in the footer, the decision is made by filter developers case-by-case. Usually, banners in the footer do not stand out against other elements and are not distracting.
 
-**What will be blocked with this filter?**
+#### AdGuard Widgets filter
 
-- This filter will block social media widgets (on third-party websites) such as "Like" and "Share" buttons, recommendation widgets, and more.
+This is a filter that blocks various widgets that are not essential for the functioning of the website or for interaction with it:
+
+- Widgets for content recommendations — related articles, similar websites, various personalized recommendations
+- Chat widgets that are not integrated with the content and are not the main content of the page
+- Marketing widgets:
+    - Chats for communication with assistants or bots
```

Comment:
How can we handle the following case?
There is a chat widget. And, the widget is used at unrelated webpage and related webpage of a website.

Should be handled with `$path` modifier?

Reply:
If annoying widget - we just block it on whole site.